main.go

```diff
@@ -37,6 +37,9 @@ func main() {
 	servercfg.SetVersion(version)
 	fmt.Println(models.RetrieveLogo()) // print the logo
 	initialize()                       // initial db and acls
+	logic.SetIpPool()
+	defer logic.ClearIpPool()
+	defer logic.WriteMetricsCacheToDB()
```

Comment:
Why are we deferring writing metrics cache to DB?
This may cause inconsistency in the data when the server quits abruptly

Reply:
done